.github/workflows/admob.yml

```diff
@@ -0,0 +1,39 @@
+name: AdMob
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    name: build
+    runs-on: macOS-latest
+    strategy:
+        matrix:
+          swift-suffix: ['', 'Swift']
+
+    steps:
```

Comment:
Also consider building and testing the test targets.

Reply:
This should be handled automatically by `scripts/test.sh`, but it looks like secrets always fail to install. Is this script intended to be used from firebase-ios-sdk's checks?

Reply:
The tests fail without credentials present due to some header stuff that's used by firebase-ios-sdk. I'll merge the PR as-is and we can pursue adding the test targets separately.

Reply:
SGTM